### PR TITLE
[CI]Suppress pip logs

### PR DESCRIPTION
**Pull request by xta0**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #32147 [iOS] Update Gemfile
* **#31912 [CI]Suppress pip logs**
### Summary

Clean up the logs from pip-install. 

### Test Plan

- Don't break the iOS simulator build

Differential Revision: [D19395526](https://our.internmc.facebook.com/intern/diff/D19395526)

### CODE REVIEW

**Comment by kostmo** [comment]
## :pill: CircleCI build failures summary and remediations
As of commit 8c00abf1:


* **1/1** failures introduced in this PR


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=8c00abf1600a8f5babac68846988289eaaf88851).

### 1 failure *not* recognized by patterns:
| Job | Step | Status |
| --- | --- | --- |
| CircleCI <sup>pytorch_ios_11_2_1_x86_64_build</sup> | <sup>Run Simulator Tests</sup> | New in PR |


---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

File: .circleci/config.yml [hunk 1]
```diff
@@ -1488,9 +1488,9 @@ jobs:
             # Temporarily pin pillow to 6.2.1 as PILLOW_VERSION is replaced by
             # _version_ in 7.0.0. Long term fix should be making changes to
             # torchvision to be compatible with both < and >= v7.0.0.
-            pip install pillow==6.2.1
+            pip install pillow==6.2.1 --quite
```
Comment (zou3519) [inline .circleci/config.yml]: I think you misspelled "quiet"

Comment (xta0) [reply .circleci/config.yml]: @zou3519 can you give a green check mark ?

Comment (zou3519) [reply .circleci/config.yml]: I don't know the consequences of this, a --quiet recently caused a timeout bug. I'll request review from the people who have more context about it.

**Comment by suo** [comment]
I'm not sure we should be suppressing info here. The only time we really look at this section of the log is when something goes wrong, and if something goes wrong we want maximum information.

**Comment by xta0** [comment]
Hi @suo @zou3519, guess my real purpose is to disable the progress bar. I've updated the diff to use `--progress-bar off` instead of `--quiet`. Let me know if you have any concerns. Thanks.

**Comment by zou3519** [comment]
What does disabling the progress bar do? Can you show a snippet of the before/after in the logs?

**Comment by xta0** [comment]
@zou3519 here is the doc regarding the progress bar - https://pip.pypa.io/en/stable/reference/pip_install/ . If you're looking for an example, you can check out https://circleci.com/gh/pytorch/pytorch/4238444?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link/console. Scroll to the bottom and click the last section to see the logs. Something like this:

```
.0MB 9.6MB/s eta 0:00:06 |██████▎ | 
14.0MB 9.6MB/s eta 0:00:06 |██████▎ | 
14.0MB 9.6MB/s eta 0:00:06 |██████▎ | 
14.0MB 9.6MB/s eta 0:00:06 |██████▎ | 
```
the `--progress-bar off` will just disable the progress bar as outputs

**Review by zou3519 — APPROVED** [review]
Sorry, I didn't realize that this was just in the ios simulator tests. Go for it!

**Pull request closed without merging**